modules/nw-egress-ips-about.adoc

```diff
@@ -18,7 +18,7 @@ The {product-title} egress IP address functionality allows you to ensure that th
 For example, you might have a pod that periodically queries a database that is hosted on a server outside of your cluster. To enforce access requirements for the server, a packet filtering device is configured to allow traffic only from specific IP addresses.
 To ensure that you can reliably allow access to the server from only that specific pod, you can configure a specific egress IP address for the pod that makes the requests to the server.
 
-An egress IP address assigned to a namespace is different from an egress router, which is used to send traffic to specific destinations.
+An egress IP address assigned to a namespace is different from an egress router, which is used to send traffic to specific destinations. When application pod and router pod are running on the same node and an egress IP is configured for the application project, the egress IP is not used.
```

Comment:
I understand here router means ingress router, and the egress IP is not used when send request to a route, so suggest updated it like this and better in a separate line to avoid confusion for egress router.

"When application pod and ingress router pod are running on the same node and an egress IP is configured for the application project, the egress IP is not used when send request to a route  from the application project which has egressIP configured."

@kyrtapz Could you help check? If I understand it wrong, please correct me, thanks!

Reply:
@huiran0826 is right, egress router is not at play here

Reply:
@huiran0826 I addressed the change